cheroot/test/test_conn.py

```diff
@@ -948,8 +984,13 @@ def test_Content_Length_out(
 
     assert actual_status == expected_resp_status
     assert actual_resp_body == expected_resp_body
-
     conn.close()
```

Comment:
I'd prefer to keep the separator because close isn't strictly related to the assertions
```suggestion

    conn.close()

```